marshmallow/fields.py

```diff
@@ -191,8 +191,12 @@ def serialize(self, obj, fields_dict, many=False, strict=False, skip_missing=Fal
                 exception_class=MarshallingError,
                 strict=strict
             )
-            if (value is missing) or (skip_missing and
-                                      value in field_obj.SKIPPABLE_VALUES):
+            skip_conds = (
+                field_obj.load_only,
```

Comment:
Actually, we can check `field_obj.load_only` before running `_call_and_store` as well.